### ISSUE TITLE
bin/console scheduler:list error

### ISSUE THREAD

**Issue by xpackal5** [contributte/scheduler#13]
If i try list command it throws exception

`A cell must be a TableCell, a scalar or an object implementing __toString, array given.` 

command run is OK

My config
```
scheduler:
    path: '%tempDir%/scheduler'
    jobs:
        - {cron: '0 1 * * *', callback: [@app\modules\User\scheduler\AuthCodeCleaner, do]}
```

**Comment by petrparolek** [comment]
It would be fixed by my commit 84c58891e5fc99c10b83726772ef39bdd95527f4 in master branch

**Comment by mabar** [comment]
Petr is right, fixed
